ex_standards/ex_http_status.md

```diff
@@ -9,34 +9,34 @@ Some might ok with 2 options, choose one and justify it.
 ----
 
 * I cannot access that web if I'm not logged:
-  *  
+  401 Unauthorized.
 
 * I'm logged but I cannot access that page
-  * 
+  403 forbidden
 
 * The item with that Id was deleted
-  * 
+  200 OK
```

Comment:
Normalmente sería un 404 (not found) ou un 410 gone se a API ten "memoria", en algunhas ocasións podemos atopar un 200 vacío